deploy-service/teletraanservice/src/main/java/com/pinterest/teletraan/config/PhabricatorFactory.java

```diff
@@ -24,6 +24,10 @@
 
 @JsonTypeName("phabricator")
 public class PhabricatorFactory implements SourceControlFactory {
+    @NotNull
+    @JsonProperty
+    private String typeName;
```

Comment:
Yes, when a GithubManager or PhabricatorManager object is instantiated it will need to have access to this Json value from the config file since it holds a specific value

Reply:
Missed that part, thanks!